vmdb/app/models/miq_report/search.rb

```diff
@@ -14,7 +14,7 @@ def get_limit_offset(page, per_page)
   end
 
   def get_sqltable(assoc)
-    r = self.db_class.reflect_on_association(assoc.to_sym)
+    r = db_class.reflection_with_virtual(assoc.to_sym)
```

Comment:
Wait no, before my change, if it was an `ActsAsArModel`, you would get back a virtual reflection.  But it `db_class` was a regular model, you would get back real reflections.  I guess to be totally backwards compatible, I'd have to add a conditional here like:

```ruby
if db_class == VmdbDatabaseConnection
  # get virtual reflections
else
  # get real reflections
end
```